unix_integration/resolver/src/lib.rs

```diff
@@ -1,4 +1,6 @@
-#![deny(warnings)]
+// Swear to god william, don't forget to uncomment this
+// #![deny(warnings)]
```

Comment:
🤔

Reply:
Don't make me tap the sign *draft PR* :P